lhotse/audio.py

```diff
@@ -14,7 +14,7 @@
         import librosa
 import numpy as np
 
-from lhotse.utils import Pathlike, SetContainingAnything, Seconds, load_yaml, save_to_yaml
+from lhotse.utils import Pathlike, Decibels, SetContainingAnything, Seconds, load_yaml, save_to_yaml
```

Comment:
Should the import be sorted alphabetically?

Reply:
Is sorting by alphabetic really needed? @pzelasko

Reply:
I don't have such strict requirements, as I actually almost never write the import lines myself - I use PyCharm's auto-import to import the relevant symbols, and then PyCharm's "optimize imports" so that it re-arranges them whatever way is appropriate. In general, I prefer not to burn too much energy on these kinds of things if it can be automated.

Reply:
Actually, feel free to reformat the code with https://github.com/psf/black if you like, I think it does re-arrange the imports too?

Reply:
Consistency matters.

If @pzelasko  uses PyCharm to sort the import automatically, which follows the alphabetic order, then the remaining
code should also follow this style.

The are various style guides recommending an alphabetic import order.

Please refer to https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

Reply:
There is a nice link there for a utility that does this automatically: https://pypi.org/project/isort/

I'm considering writing a dev-helper script at some point that runs stuff like this + black + mypy to address consistency issues.

Reply:
PR #58.